__tests__/SelfMailersApi.unit.ts

```diff
@@ -518,7 +513,7 @@ describe("SelfMailersApi", () => {
         undefined,
         undefined,
         undefined,
-        new SendDate(DATE_FILTER)
```

Comment:
oh thank you, I really disliked this for some reason

Reply:
It was such a weird piece of code lol